mode-apis-base/src/alertingModels.ts

```diff
@@ -180,6 +215,7 @@ export interface AlertRule {
     readonly description?: string | undefined;
     readonly enabled: boolean;
     readonly condition: AlertCondition;
+    readonly messageTemplate: AlertMessageTemplate;
```

Comment:
I assume this information will not only be used in notification/action but also used in creating Alert.